docs/src/components/productBaseUI/MuiBaseDeprecation.tsx

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable no-irregular-whitespace */
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import { MarkdownElement } from '@mui/docs/MarkdownElement';
+
+export default function MuiBaseDeprecation(props: {
+  newComponentName?: string;
+  newComponentUrl?: string;
+}) {
+  if (props.newComponentUrl && props.newComponentName) {
+    return (
+      <MarkdownElement>
+        <Box component="aside" className="MuiCallout-root MuiCallout-error">
+          <Icon />
+          <Box className="MuiCallout-content">
+            @mui/base has been deprecated and has been replaced by Base UI. Please use the Base UI{' '}
+            <a href={props.newComponentUrl}>{props.newComponentName} component</a> instead.
+          </Box>
+        </Box>
+      </MarkdownElement>
+    );
+  }
+  return (
+    <MarkdownElement>
+      <Box component="aside" className="MuiCallout-root MuiCallout-error">
+        <Icon />
+        <Box className="MuiCallout-content">
+          @mui/base has been deprecated and has been replaced by{' '}
+          <a href="https://www.base-ui.com">Base UI</a>. We strongly advise using the new package
```

Comment:
No double redirection, Google doesn't like that, and it's frustrating for users (slower). Fixing this in: #45146.